src/org/myworldgis/netlogo/ProjectLatLon.java

```diff
@@ -0,0 +1,127 @@
+package org.myworldgis.netlogo;
+
+import org.nlogo.api.Argument;
+import org.nlogo.api.Context;
+import org.nlogo.api.ExtensionException;
+import org.nlogo.api.LogoException;
+import org.nlogo.api.LogoListBuilder;
+import org.nlogo.core.Syntax;
+import org.nlogo.core.SyntaxJ;
+import org.myworldgis.projection.Ellipsoid;
+import org.myworldgis.projection.Geographic;
+import org.myworldgis.projection.Projection;
+import org.ngs.ngunits.NonSI;
+import org.ngs.ngunits.SI;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.util.GeometryTransformer;
+
+
+/**
+ * 
+ */
+public abstract strictfp class ProjectLatLon {
+
+    private static final Ellipsoid WGS84 = new Ellipsoid("WGS 84", 6378137.0, SI.METER, 298.257223563);
```

Comment:
Should this class use the same static `WGS_84` that is already in the `Ellipsoid` class, rather than defining its own?